src/library.js

```diff
@@ -448,16 +448,16 @@ LibraryManager.library = {
     }
     return ret;  // Previous break location.
   },
-
   system__deps: ['__setErrNo', '$ERRNO_CODES'],
-  system: function(command) {
-    // int system(const char *command);
-    // http://pubs.opengroup.org/onlinepubs/000095399/functions/system.html
-    // Can't call external programs.
-    ___setErrNo(ERRNO_CODES.EAGAIN);
-    return -1;
+  system: function(command) 
```

Comment:
ok i can repost you a code align ,with emscripten guide lines
{ in the same lines, or may you change directly code ?
 without other pull request ?